extensions/amp-sticky-ad/amp-sticky-ad.md

```diff
@@ -62,3 +63,9 @@ Example:
 **layout**
 
 The only permissible value for the `layout` attribute in `amp-sticky-ad` is `nodisplay`.
+
```

Comment:
Recommend adding a note on "Orientation" : 
"Adjusting to landscape mode, will center align the ad fluidly."